Add minimum requirement content (#3)
kubernetes-i18n-ukrainian/website: content/uk/docs/tutorials/hello-minikube.md

    {{< note >}}Якщо Minikube встановлений локально, виконайте команду `minikube start`.{{< /note >}}

<!--2. Open the Kubernetes dashboard in a browser:
-->
2. Відкрийте Kubernetes dashboard у браузері:

    ```shell
    minikube dashboard
    ```

<!--3. Katacoda environment only: At the top of the terminal pane, click the plus sign, and then click **Select port to view on Host 1**.
-->
3. Тільки для Katacoda: у верхній частині вікна термінала натисніть знак плюс, а потім -- **Select port to view on Host 1**.

<!--4. Katacoda environment only: Type `30000`, and then click **Display Port**.
-->
4. Тільки для Katacoda: введіть `30000`, а потім натисніть **Display Port**. 

<!--## Create a Deployment
-->
## Створення Deployment

<!--A Kubernetes [*Pod*](/docs/concepts/workloads/pods/pod/) is a group of one or more Containers,
tied together for the purposes of administration and networking. The Pod in this
tutorial has only one Container. A Kubernetes
[*Deployment*](/docs/concepts/workloads/controllers/deployment/) checks on the health of your
Pod and restarts the Pod's Container if it terminates. Deployments are the
recommended way to manage the creation and scaling of Pods.
-->

> [*Под*](/docs/concepts/workloads/pods/pod/) у Kubernetes -- це група з одного або декількох контейнерів, що об'єднані разом з метою адміністрування і роботи у мережі. У цьому навчальному матеріалі Под має лише один контейнер. Kubernetes [*Deployment*](/docs/concepts/workloads/controllers/deployment/) перевіряє стан Поду і перезапускає контейнер в Поді, якщо контейнер перестає працювати. Створювати і масштабувати Поди рекомендується за допомогою Deployment'ів.

Author: ```suggestion
[*Под*](/docs/concepts/workloads/pods/pod/) у Kubernetes -- це група з одного або декількох контейнерів, що об'єднані разом з метою адміністрування і роботи у мережі. У цьому навчальному матеріалі Под має лише один контейнер. Kubernetes [*Deployment*](/docs/concepts/workloads/controllers/deployment/) перевіряє стан Поду і перезапускає контейнер Пода, якщо контейнер перестає працювати. Створювати і масштабувати Поди рекомендується за допомогою Deployment'ів.
```

<!--1. Use the `kubectl create` command to create a Deployment that manages a Pod. The
Pod runs a Container based on the provided Docker image.
-->
1. За допомогою команди `kubectl create` створіть Deployment, який керуватиме Подом. Под запускає контейнер на основі наданого Docker образу.

    ```shell
    kubectl create deployment hello-node --image=gcr.io/hello-minikube-zero-install/hello-node
    ```

<!--2. View the Deployment:
-->

> 2. Переглянути інформацію про запущений Deployment:

Author: ```suggestion
2. Перегляньте інформацію про запущений Deployment:
```

    ```shell
    kubectl get deployments
    ```

    <!--The output is similar to:
    -->
    У виводі ви побачите подібну інформацію:

    ```
    NAME         READY   UP-TO-DATE   AVAILABLE   AGE
    hello-node   1/1     1            1           1m
    ```

<!--3. View the Pod:
-->

> 3. Переглянути інформацію про запущені Поди:

Author: ```suggestion
3. Перегляньте інформацію про запущені Поди:
```

> 3. Переглянути інформацію про под:

Reviewer: Я схиляюсь до інфінітиву.

-->
1. Відкрийте под для загального доступу в інтернет за допомогою команди `kubectl expose`:

    ```shell
    kubectl expose deployment hello-node --type=LoadBalancer --port=8080
    ```
    
    <!--The `--type=LoadBalancer` flag indicates that you want to expose your Service
    outside of the cluster.
    -->
    Флаг `--type=LoadBalancer` означає, що ви хочете відкрити сервіс за межі кластера.

<!--2. View the Service you just created:
-->
2. Перегляньте інформацію за сервісом, який ви щойно створили:

    ```shell
    kubectl get services
    ```

    <!--The output is similar to:
    -->
    У виводі ви побачите подібну інформацію:

    ```
    NAME         TYPE           CLUSTER-IP      EXTERNAL-IP   PORT(S)          AGE
    hello-node   LoadBalancer   10.108.144.78   <pending>     8080:30369/TCP   21s
    kubernetes   ClusterIP      10.96.0.1       <none>        443/TCP          23m
    ```

    <!--On cloud providers that support load balancers,
    an external IP address would be provisioned to access the Service. On Minikube,
    the `LoadBalancer` type makes the Service accessible through the `minikube service`
    command.
    -->

>     Для хмарних провайдерів, що підтримують балансування навантаження: доступ до сервісу надається через зовнішню IP-адресу. Для Minikube: параметр тип `LoadBalancer` відкриває сервіс через команду `minikube service`.

Author: ```suggestion
    Для хмарних провайдерів, що підтримують балансування навантаження, доступ до сервісу надається через зовнішню IP-адресу. Для Minikube, тип `LoadBalancer` робить Сервіс доступним ззовні за допомогою команди `minikube service`.
```

<!--3. Run the following command:
-->

> 3. Запустіть наступну команду:

Author: ```suggestion
3. Виконайте наступну команду:
```

    ```shell
    minikube service hello-node
    ```

<!--4. Katacoda environment only: Click the plus sign, and then click **Select port to view on Host 1**.
-->

> 4. Лише для середовища Katacoda: натисніть знак плюс, а потім -- **Select port to view on Host 1 (Обрати порт для відображення на хості 1)**.

Author: ```suggestion
4. Тільки для Katacoda: натисніть знак плюс, а потім - **Select port to view on Host 1**.
```

<!--5. Katacoda environment only: Note the 5 digit port number displayed opposite to `8080` in services output. This port number is randomly generated and it can be different for you. Type your number in the port number text box, then click Display Port. Using the example from earlier, you would type `30369`.
-->

> 5. Лише для середовища Katacoda: запишіть п'ятизначний номер порту, що відображається напроти `8080` у виводі даних сервісу. Номер цього порту генерується у довільному порядку і може бути іншим у вашому випадку. Введіть номер порту у призначене для цього текстове поле і натисніть Display Port (Відобразити порт). У нашому прикладі номер порту `30369`.

Author: ```suggestion
5. Тільки для Katacoda: запишіть п'ятизначний номер порту, що відображається напроти `8080` у виводі сервісу. Номер цього порту генерується довільно і тому може бути іншим у вашому випадку. Введіть номер порту у призначене для цього текстове поле і натисніть Display Port. У нашому прикладі номер порту `30369`.
```

    <!--This opens up a browser window that serves your app and shows the "Hello World" message.
    -->

>     Це відкриє вікно браузера, в якому запущений ваш застосунок, і відобразить повідомлення "Hello World".

Author: ```suggestion
    Це відкриє вікно браузера, в якому запущений ваш застосунок, і покаже повідомлення "Hello World".
```

<!--## Enable addons
-->

> ## Підтримка розширень

Author: ```suggestion
## Увімкнення розширень
```

<!--Minikube has a set of built-in {{< glossary_tooltip text="addons" term_id="addons" >}} that can be enabled, disabled and opened in the local Kubernetes environment.
-->

> Minikube має ряд вбудованих {{< glossary_tooltip text="розширень" term_id="addons" >}}, які можна активувати, дезактивувати і відкрити у локальному Kubernetes оточенні.

Author: ```suggestion
Minikube має ряд вбудованих {{< glossary_tooltip text="розширень" term_id="addons" >}}, які можна увімкнути, вимкнути і відкрити у локальному Kubernetes оточенні.
```

<!--1. List the currently supported addons:
-->

> 1. Переглянути перелік підтримуваних розширень:

Author: ```suggestion
1. Перегляньте перелік підтримуваних розширень:
```

    ```shell
    minikube addons list
    ```

    <!--The output is similar to:
    -->
    У виводі ви побачите подібну інформацію:

    ```
    addon-manager: enabled
    dashboard: enabled
    default-storageclass: enabled
    efk: disabled
    freshpod: disabled
    gvisor: disabled
    helm-tiller: disabled
    ingress: disabled
    ingress-dns: disabled
    logviewer: disabled
    metrics-server: disabled
    nvidia-driver-installer: disabled
    nvidia-gpu-device-plugin: disabled
    registry: disabled
    registry-creds: disabled
    storage-provisioner: enabled
    storage-provisioner-gluster: disabled
    ```
   
<!--2. Enable an addon, for example, `metrics-server`:
-->

> 2. Увімкнути розширення, наприклад `metrics-server`:

Author: ```suggestion
2. Увімкніть розширення, наприклад `metrics-server`:
```

> 2. Активувати розширення, наприклад `metrics-server`:

Author: ```suggestion
2. Увімкніть розширення, наприклад `metrics-server`:
```

    ```shell
    minikube addons enable metrics-server
    ```
  
    <!--The output is similar to:
    -->
    У виводі ви побачите подібну інформацію:

    ```
    metrics-server was successfully enabled
    ```

<!--3. View the Pod and Service you just created:
-->

> 3. Переглянути інформацію про Под і Сервіс, які ви щойно створили:

Author: ```suggestion
3. Перегляньте інформацію про Под і Сервіс, які ви щойно створили:
```

    pod/influxdb-grafana-b29w8                  2/2       Running   0          26s
    pod/kube-addon-manager-minikube             1/1       Running   0          34m
    pod/kube-apiserver-minikube                 1/1       Running   0          34m
    pod/kube-controller-manager-minikube        1/1       Running   0          34m
    pod/kube-proxy-rnlps                        1/1       Running   0          34m
    pod/kube-scheduler-minikube                 1/1       Running   0          34m
    pod/storage-provisioner                     1/1       Running   0          34m

    NAME                           TYPE        CLUSTER-IP      EXTERNAL-IP   PORT(S)             AGE
    service/metrics-server         ClusterIP   10.96.241.45    <none>        80/TCP              26s
    service/kube-dns               ClusterIP   10.96.0.10      <none>        53/UDP,53/TCP       34m
    service/monitoring-grafana     NodePort    10.99.24.54     <none>        80:30002/TCP        26s
    service/monitoring-influxdb    ClusterIP   10.111.169.94   <none>        8083/TCP,8086/TCP   26s
    ```

<!--4. Disable `metrics-server`:
-->

> 4. Вимкнути `metrics-server`:

Author: ```suggestion
4. Вимкніть `metrics-server`:
```
